marienfressinaud/FreshRSS
# Delete jQuery

Closes #3845

Changes proposed in this pull request:
- delete jquery.min.js
- delete lib_phpQuery


jQuery is included just in the stats, but it looks like that we does not need it anymore.
So in general jQuery is not needed anymore in the core.

lib_phpQuery looks like unused in the core. 
Also: It looks like that the development and active usage of phpQuery stopped in 2015: https://code.google.com/archive/p/phpquery/issues/246


How to test the feature manually:
1. check the stats (because of jQuery)
2. and check the system in general (because of phpQuery)

Pull request checklist:


I will publish this changes as draft first! Will see if the changes breaks the system or not.
